tests/isolated/regular_file_readall.py

```diff
@@ -0,0 +1,42 @@
+import eventlet
+eventlet.monkey_patch()
```

Comment:
Please, follow this pattern for isolated tests:
```
__test__ = False

if __name__ == '__main__':
    import eventlet
    # and everything else
```

`monkey_patch()` at module level is a hidden bomb in a file that is potentially imported by test runner collector.